scripts/extensions/videoEditor/src/VideoTimeline/VideoTimeline.tsx

```diff
@@ -235,11 +225,23 @@ export class VideoTimeline extends React.Component<IProps, IState> {
         const video = this.props.video;
         const left = video ? `${(this.state.trim.start / video.duration) * 100}%` : '0%';
         const right = video ? `${(1 - this.state.trim.end / video.duration) * 100}%` : '0%';
+        const thumbnailWidth = Math.round(video.videoWidth / (video.videoHeight / 50)); // formula from video server
+        // maximum total possible thumbnails, minus margin to avoid making it feels too narrow
+        const totalThumbnails = Math.round(((this.timeline.current?.offsetWidth ?? 0) - 40 * 2) / thumbnailWidth);
```

Comment:
Can you avoid duplicating these margin numbers -  `40 * 2`?. Since it doesn't have any connection to the original definition it will render incorrectly if the original number. Instead of duplicating the numbers, could you add a wrapper div? Or at least read the margin from an element so it would stay in sync if the margin was updated?

Reply:
Ah my bad, forgot I could get this via `getComputedStyle`, and this is more like parent `padding` than margin